e2e/docker-image/docker-image.test.ts

```diff
@@ -0,0 +1,32 @@
+import { createTenv } from '@e2e/tenv';
+
+const { container } = createTenv(__dirname);
+
+beforeAll(async () => {
+  const isCI = !!process.env.CI;
+  const image = isCI ? 'ghcr.io/ardatan/graphql-mesh:test' : 'graphql-mesh';
+
+  container({
+    name: 'graphql-mesh-test',
+    image,
+    containerPort: 4000,
+    hostPort: 4000,
+    healthcheck: [
+      'CMD-SHELL',
+      'echo "this healthcheck is weirdly stuck on checking when http://0.0.0.0:4000 is ready, so this is to bypass it"',
+    ],
+    pipeLogs: true,
+    skipImagePulling: true,
+  });
+
+  // wait for container to be ready
+  await new Promise(res => setTimeout(res, 8000));
```

Comment:
https://github.com/ardatan/graphql-mesh/blob/6aa3fbf28598c88de3ca10bc3426e66f371bb945/e2e/utils/tenv.ts#L379-L392

```suggestion
await container({
    name: 'graphql-mesh-test',
    image,
    containerPort: 4000,
    hostPort: 4000,
    healthcheck: [
      'CMD-SHELL',
      'echo "this healthcheck is weirdly stuck on checking when http://0.0.0.0:4000 is ready, so this is to bypass it"',
    ],
    pipeLogs: true,
    skipImagePulling: true,
  });
```

Reply:
the `await` goes on forever